pom.xml

```diff
@@ -5145,13 +5146,14 @@
       </dependency>
       <dependency>
         <groupId>org.springframework</groupId>
-        <artifactId>spring-webmvc-portlet</artifactId>
+        <artifactId>spring-websocket</artifactId>
         <version>${version.org.springframework}</version>
       </dependency>
+
       <dependency>
-        <groupId>org.springframework</groupId>
-        <artifactId>spring-websocket</artifactId>
-        <version>${version.org.springframework}</version>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${version.org.springframework.boot}</version>
```

Comment:
Let's remove this whole dependency. I've checked and I couldn't import the BOM without specifying version so this actually doesn't work as I expected.

Reply:
But we can do that in another PR. This has been open for too long.

Reply:
Noted. I'll make a Jira for this.